src/pages/image/_id.vue

```diff
@@ -1,48 +1,127 @@
+
+    <section
+      id="title-button"
+      class="flex flex-row md:flex-row-reverse flex-wrap justify-between md:mt-8"
+    >
+      <VButton
+        as="a"
+        :href="image.foreign_landing_url"
+        target="blank"
+        rel="noopener noreferrer"
+        class="btn-main flex-initial w-full md:w-max mb-4 md:mb-0"
+        size="large"
+        @click="onSourceLinkClicked"
+        @keyup.enter="onSourceLinkClicked"
+        >{{ $t('image-details.weblink') }}</VButton
+      >
+      <span class="flex-1">
+        <h1 class="text-base md:text-3xl font-semibold leading-[130%]">
+          {{ image.title }}
+        </h1>
+        <i18n
+          v-if="image.creator"
+          path="image-details.creator"
+          tag="span"
+          class="font-semibold leading-[130%]"
+        >
+          <template #name>
+            <a
+              v-if="image.creator_url"
+              :aria-label="
+                $t('media-details.aria.creator-url', {
+                  creator: image.creator,
+                })
+              "
+              :href="image.creator_url"
+              target="blank"
+              rel="noopener noreferrer"
+              @click="onCreatorLinkClicked"
+              @keyup.enter="onCreatorLinkClicked"
+              >{{ image.creator }}</a
+            >
+            <span v-else>{{ image.creator }}</span>
+          </template>
+        </i18n>
+      </span>
+    </section>
+
+    <VMediaReuse :media="image" />
+    <VImageDetails
       :image="image"
-      :thumbnail="thumbnailURL"
-      :bread-crumb-u-r-l="breadCrumbURL"
-      :should-show-breadcrumb="shouldShowBreadcrumb"
       :image-width="imageWidth"
       :image-height="imageHeight"
       :image-type="imageType"
-      @onImageLoaded="onImageLoaded"
     />
-    <RelatedImages :image-id="imageId" />
+    <VRelatedImages :image-id="imageId" />
   </div>
 </template>
 
 <script>
 import axios from 'axios'
 import { mapActions, mapState } from 'vuex'
 import { FETCH_IMAGE } from '~/constants/action-types'
-import { MEDIA } from '~/constants/store-modules'
-import RelatedImages from '~/components/ImageDetails/RelatedImages.vue'
-import VPhotoDetails from '~/components/ImageDetails/VPhotoDetails.vue'
+import { MEDIA, USAGE_DATA } from '~/constants/store-modules'
+import {
+  DETAIL_PAGE_EVENTS,
+  SEND_DETAIL_PAGE_EVENT,
+} from '~/constants/usage-data-analytics-types'
+
+import VImageDetails from '~/components/VImageDetails/VImageDetails.vue'
+import VRelatedImages from '~/components/VImageDetails/VRelatedImages.vue'
+import VMediaReuse from '~/components/VMediaInfo/VMediaReuse.vue'
+import VButton from '~/components/VButton.vue'
+import VIcon from '~/components/VIcon/VIcon.vue'
 
-const PhotoDetailPage = {
-  name: 'PhotoDetailPage',
-  components: { RelatedImages, VPhotoDetails },
+import Chevron from '~/assets/icons/chevron-left.svg?inline'
+
+const VImageDetailsPage = {
+  name: 'VImageDetailsPage',
+  layout: 'blank',
```

Comment:
I'Using the `blank` layout is causing RTL to break. Is this just temporary to remove the header for reviewing and we'll switch back to `default` after?

Reply:
Yes, it was a temporary workaround to show the page without the horizontal scroll that the header was causing but it's already fixed so now it's back to the default layout.